pupil_src/shared_modules/glfw.py

```diff
@@ -424,6 +424,7 @@ class GLFWmonitor(Structure):
 glfwGetPrimaryMonitor = _glfw.glfwGetPrimaryMonitor
 glfwGetPrimaryMonitor.restype = POINTER(GLFWmonitor)
 # glfwGetMonitorPos               = _glfw.glfwGetMonitorPos
+# glfwGetMonitorWorkarea          = _glfw.glfwGetMonitorWorkarea
```

Comment:
Is this commented out on purpose?

Reply:
Yes; added only for consistency with other definitions.